src/shared/models/supportcategorymodel.cpp

```diff
@@ -9,41 +9,33 @@
 #include "logger.h"
 
 namespace {
+SupportCategoryModel* s_instance = nullptr;
 Logger logger("SupportCategoryModel");
 
 struct SupportCategory {
   const char* m_categoryName;
-  I18nStrings::String m_stringId;
 };
 
 static QList<SupportCategory> s_supportCategories{
-    SupportCategory{"payment",
-                    I18nStrings::InAppSupportWorkflowPaymentDropdownListItem},
-    SupportCategory{"account",
-                    I18nStrings::InAppSupportWorkflowAccountDropdownListItem},
-    SupportCategory{"technical",
-                    I18nStrings::InAppSupportWorkflowTechnicaDropdownListItem},
-    SupportCategory{
-        "feature",
-        I18nStrings::InAppSupportWorkflowFeatureRequestDropdownListItem},
-    SupportCategory{"other",
-                    I18nStrings::InAppSupportWorkflowOtherDropdownListItem},
+    SupportCategory{"InAppSupportWorkflowPaymentDropdownListItem"},
+    SupportCategory{"InAppSupportWorkflowAccountDropdownListItem"},
+    SupportCategory{"InAppSupportWorkflowTechnicaDropdownListItem"},
+    SupportCategory{"InAppSupportWorkflowFeatureRequestDropdownListItem"},
+    SupportCategory{"InAppSupportWorkflowOtherDropdownListItem"},
 };
 
 }  // namespace
 
-SupportCategoryModel::SupportCategoryModel() {
-  MZ_COUNT_CTOR(SupportCategoryModel);
-}
-
-SupportCategoryModel::~SupportCategoryModel() {
-  MZ_COUNT_DTOR(SupportCategoryModel);
+SupportCategoryModel* SupportCategoryModel::instance() {
+  if (!s_instance) {
+    s_instance = new SupportCategoryModel();
```

Comment:
you are probably going to leak this object. The fix for this is:

1. the CTOR should be: `explicit SupportCategoryModel(QObject* parent)`
2. the CTOR impl should be: `SupportCategoryModel::SupportCategoryModel(QObject* parent) : QAbstractModelList(parent) { ...`
3. in here, call: `s_instance = new SupportCategoryModel(qApp);`
4. include `QCoreApplication` to have access to `qApp`